File: src/main/java/br/com/jiratorio/domain/changelog/JiraChangelogHistory.java
package br.com.jiratorio.domain.changelog;

import com.fasterxml.jackson.annotation.JsonFormat;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import java.time.LocalDateTime;
import java.util.List;
import lombok.Data;

@Data
@JsonIgnoreProperties(ignoreUnknown = true)
public class JiraChangelogHistory {

    private String id;

    private JiraChangelogAuthor author;

    @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSZ")
    private LocalDateTime created;

    private List<JiraChangelogItem> items;

}

File: src/main/kotlin/br/com/jiratorio/domain/jira/changelog/JiraChangelogHistory.kt
package br.com.jiratorio.domain.jira.changelog

import com.fasterxml.jackson.annotation.JsonFormat
import java.time.LocalDateTime

data class JiraChangelogHistory(

    val id: String,

    val author: JiraChangelogAuthor,

    @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSZ")
    val created: LocalDateTime,

    val items: List<JiraChangelogItem>

)
